open-company/open-company-web
# Update page title

Card: https://trello.com/c/akbZDo9W

To test:
- open marketing site and check all the pages: /, /slack, /pricing, /terms, /privacy
- [x] do you see the updated default title?
- now nav to an org page
- [x] do you see the other title: `Carrot | Company name`
